code/stable-website/index.html

```diff
@@ -236,24 +236,24 @@ <h1>Generated Examples</h1>
             </ul>
           </li>
           <li>
-            PD_Controller  - To provide a model of a PD Controller that can be used for the tuning of the gain constants before the deployment of the controller.
+            PD Controller  - To provide a model of a PD Controller that can be used for the tuning of the gain constants before the deployment of the controller.
             <ul class="list">
               <li>
-                PD_Controller_SRS <a href="examples/pd_controller/SRS/srs/PD_Controller_SRS.html">[HTML]</a> <a href="examples/pd_controller/SRS/srs/PD_Controller_SRS.pdf">[PDF]</a>
+                SRS: <a href="examples/pdcontroller/SRS/srs/PDController_SRS.html">[HTML]</a> <a href="examples/pdcontroller/SRS/srs/PDController_SRS.pdf">[PDF]</a>
               </li>
               <li>
                 Generated Code:
                 <ul class="list">
                   <li>
-                    PD_Controller <a href="https://github.com/JacquesCarette/Drasil/tree/master/code/stable/pd_controller/src/python">[Python]</a> <a href="https://github.com/JacquesCarette/Drasil/tree/master/code/stable/pd_controller/src/csharp">[C Sharp]</a> <a href="https://github.com/JacquesCarette/Drasil/tree/master/code/stable/pd_controller/src/java">[Java]</a> <a href="https://github.com/JacquesCarette/Drasil/tree/master/code/stable/pd_controller/src/cpp">[C++]</a>
+                    PDController <a href="https://github.com/JacquesCarette/Drasil/tree/master/code/stable/pdcontroller/src/python">[Python]</a> <a href="https://github.com/JacquesCarette/Drasil/tree/master/code/stable/pdcontroller/src/csharp">[C Sharp]</a> <a href="https://github.com/JacquesCarette/Drasil/tree/master/code/stable/pdcontroller/src/java">[Java]</a> <a href="https://github.com/JacquesCarette/Drasil/tree/master/code/stable/pdcontroller/src/cpp">[C++]</a>
```

Comment:
Here is the resultant 'actual' fix. Note that the name shown here doesn't align with the name shown on line 239!

Why? Well, the top-level `<li>` nodes show the main abbreviation, but this is the name of the final 'executable program/project/case study, taking into consideration the choices'. Following in line with how the other projects (including the multi-src Projectile case study in mind), we use the name of the 'executable program', left uppercase. 

If we wanted to make this name lowercase, we can, similar to how this name is done everywhere else. If we wanted to avoid putting the name of the case study here again, we can (I would prefer this, but I thought I was pushing the boundary of this PR too much now). If we want to include the space, we can. Ultimately, I didn't quite know what name we wanted here, but finally, it notes the 'real' issue here:

We have impromptu naming and calculation of things everywhere. This PR is specifically about the name of `PD Controller` and how we propagate it, but us having the freedom of Haskell is a bit of a double-edged sword here, whereby we can have impromptu temporary code placed anywhere that breaks uniformity and actual knowledge captured for a specific 'fix'.